pkg/kvstore/etcd.go

```diff
@@ -457,6 +478,7 @@ func (e *etcdClient) LockPath(ctx context.Context, path string) (kvLocker, error
 
 func (e *etcdClient) DeletePrefix(path string) error {
 	duration := spanstat.Start()
+	e.limiter.Wait(ctx.TODO())
```

Comment:
What is the need to change from `ctx.Background()` to `ctx.TODO()`?

Reply:
TODO() documents that the context is ommited because it is not available. Background() documents that the background context should be used. We would specify the context if it was available in all these cases so we should use TODO()